caffe2/contrib/tensorrt/trt_utils.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include "caffe2/core/logging.h"
+#include <NvInfer.h>
+#include <iostream>
+
+namespace caffe2 {
+
+  // Logger for GIE info/warning/errors
+class TrtLogger : public nvinfer1::ILogger {
+  using nvinfer1::ILogger::Severity;
+
+ public:
+  TrtLogger(
+      Severity verbosity = Severity::kWARNING,
+      std::ostream& ostream = std::cout)
+      : _verbosity(verbosity), _ostream(&ostream) {}
+  void log(Severity severity, const char* msg) override {
```

Comment:
Please forward it to Caffe2/GLOG logger instead, e.g. `LOG(INFO/WARNING/ERROR) << ...`